### PR TITLE
Add Generic Sensor API.

### PR DESCRIPTION
**Pull request by jpmedley**

### CODE REVIEW

File: api/AmbientLightSensor.json [hunk 1]
```diff
@@ -104,20 +168,26 @@
...
-            "webview_android": {
-              "version_added": "54"
-            }
+            "webview_android": [
+              {
+                "version_added": "54"
+              },
+              {
+                "version_added": "69",
+                "notes": ["Based on Generic Sensor API."]
```
Comment (ExE-Boss) [inline api/AmbientLightSensor.json]: This should either be:
```json
"notes": "Based on Generic Sensor API."
```
or
```json
"notes": [
  "Based on Generic Sensor API."
]
```

File: api/Sensor.json [hunk 2]
```diff
@@ -0,0 +1,464 @@
...
+            },
+            "firefox_android": {
+              "version_added": null
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": "50",
+                "version_added": "56"
```
Comment (ExE-Boss) [inline api/Sensor.json]: This should be:
```json
"opera": {
  "version_added": "56"
}
```

**Comment by jpmedley** [comment]
PTAL

**Comment by jpmedley** [comment]
@Elchi3 Actually, I added only what's implemented in Chrome. As far as I know, we're the first browser to implement. I'd prefer to wait until there's an implementation in case the spec changes. 

https://cs.chromium.org/chromium/src/third_party/blink/renderer/modules/sensor/?g=0

**Comment by Elchi3** [comment]
That makes total sense to me. Then it is just the inline comments and this should be good to go.

**Comment by jpmedley** [comment]
PTAL

**Pull request merged**